python/PiFinder/main.py

```diff
@@ -380,6 +380,7 @@ def main(
                 object_types=cfg.get_option("filter.object_types"),
                 altitude=cfg.get_option("filter.altitude", -1),
                 observed=cfg.get_option("filter.observed", "Any"),
+                selected_catalogs=cfg.get_option("active_catalogs"),
```

Comment:
I'd like to refactor this to be `filter.selected_catalogs` but the existing default.json and people's config.json have always been active_catalogs and I was a bit shy about changing too much here as part of a bug fix.

I'll merge this and do a quick refactor, it might only be a couple of little changes and worst case, people return to the default selected catalogs after update 🤷‍♂️